### DISCUSSION TITLE
Can we use this for iOS simulators as well?

### DISCUSSION THREAD

**Discussion by denil-ct** [microsoft/HydraLab#561] [General]
I couldn't get it to work with iOS simulators. Am I missing something, or is it not supported?

**Comment by hydraxman** [comment]
Could you share your environment info? For iOS we support XCTest and Appium, and here is the wiki for XCTest: https://github.com/microsoft/HydraLab/wiki/Create-test-build-and-run-XCTest

**Comment by denil-ct** [comment]
> Choose the target machine (we only support real devices for now), and run the test

Missed this line. Is support for simulators planned?

We are using a mix of XCTest and Maestro for now.

**Reply by hydraxman** [reply]
It's not in our plan before as we don't see demand for it previously. @zhou9584 and @lifesaver0129 could you guys help clarify whether we can support simulators? Is the cost high?
